libcudacxx/test/libcudacxx/std/utilities/function.objects/refwrap/binder_typedefs.compile.pass.cpp

```diff
@@ -17,7 +17,9 @@
 
 // check that binder typedefs exit
 
-// #include <cuda/std/functional>
+#define _LIBCUDACXX_DISABLE_DEPRECATION_WARNINGS
```

Comment:
suggestion: To assist future maintenance, add a comment saying what deprecated APIs this is suppressing.

Or better yet, use a more fine-grained self-documenting warning suppression if it exists (e.g. `_LIBCUDACXX_IGNORE_DEPRECATED_[SOME_API]`), but this is probably beyond the scope of this PR.